optuna/integration/_lightgbm_tuner/alias.py

```diff
@@ -57,6 +57,50 @@ def _handling_alias_parameters(lgbm_params: Dict[str, Any]) -> None:
         "metric_name": "l2",
         "alias_names": ["regression", "regression_l2", "l2", "mean_squared_error", "mse"],
     },
+    {
+        "metric_name": "l1",
+        "alias_names": ["regression_l1", "l1", "mean_absolute_error", "mae"],
+    },
+    {
+        "metric_name": "binary_logloss",
+        "alias_names": ["binary_logloss", "binary"],
+    },
+    {
+        "metric_name": "multi_logloss",
+        "alias_names": [
+            "multi_logloss",
+            "multiclass",
+            "softmax",
+            "multiclassova",
+            "multiclass_ova",
+            "ova",
+            "ovr",
+        ],
+    },
+    {
+        "metric_name": "cross_entropy",
+        "alias_names": ["cross_entropy", "xentropy"],
+    },
+    {
+        "metric_name": "cross_entropy_lambda",
+        "alias_names": ["cross_entropy_lambda", "xentlambda"],
+    },
+    {
+        "metric_name": "kullback_leibler",
+        "alias_names": ["kullback_leibler", "kldiv"],
+    },
+    {
+        "metric_name": "mape",
+        "alias_names": ["mape", "mean_absolute_percentage_error"],
+    },
+    {
+        "metric_name": "auc_mu",
+        "alias_names": ["auc_mu"],
```

Comment:
If I understand correctly, the value of `metric_name` itself should not be included in `alias_names`. The same can be said for the other metric such as "l2".

Here are the function that process alias in `LightGBMTuner `.
https://github.com/optuna/optuna/blob/master/optuna/integration/_lightgbm_tuner/alias.py#L67

Reply:
Thank you for the comment! I agree that the `metric_name` itself can be (but not necessarily) removed from the list of `alias_names`.
For the `metric_name` "auc_mu" where `alias_names` is unique and identical, I felt it's more explicit that `_ALIAS_METRIC_LIST` does cover the metric, hence added those 4 lines (but do not have effect until other alias will be added in future, which may be your point). 
Please kindly let me know your thought.

Reply:
In my opinion, it's better not to write something that isn't effective at that time. However, I'm sure each organization has its own policies, and it seems best to follow them.

Reply:
@upura 
> the value of `metric_name` itself do not have to be included in `alias_names`. The same can be said for the other metric such as "l2".

Thank you for pointing it out. The original code such as `ndcg` and `map ` by @smly does not contain `metric_name` in `alias_names`. Maybe I overlooked the inclusion of unnecessary items during the review process. 

@nyanhi 
Is it possible to remove such metric names from `alias_names`? If you do not have time enough, I'll fix it in the follow-up PR.

> "auc_mu"

I think you kindly included this metric because it is included in the list in https://github.com/optuna/optuna/issues/1806. I'm sorry for bothering you, but could you remove it? I feel it is verbose since it does not have any aliases.

Reply:
@toshihikoyanase 
Understood your points and made the changes. I have added below comment to avoid shake in notation going forward.
Thank you for the comment.
```
# The list `alias_names` do not include the `metric_name` itself.
```